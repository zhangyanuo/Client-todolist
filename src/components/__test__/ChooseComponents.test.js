import ChooseComponent from '../ChooseComponent';
import React from 'react';
import {shallow} from 'enzyme';

test('render choose component',()=>{
    const wrapper=shallow(
        <ChooseComponent/>
    )
    expect(wrapper).toMatchSnapshot();
})