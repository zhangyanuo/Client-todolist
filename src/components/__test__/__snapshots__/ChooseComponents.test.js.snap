// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render choose component 1`] = `ShallowWrapper {}`;
